src/foundation.cs

```diff
@@ -6092,6 +6913,33 @@ partial interface NSUrlSessionTask : NSCopying {
 		[Export ("priority")]
 		float Priority { get; set; } /* float, not CGFloat */
 #endif
+
+#if MONOMAC // Introspection fails for iOS
+		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
+		[Export ("progress", ArgumentSemantic.Strong)]
+		NSProgress Progress { get; }
+#endif
+
+//1) ApiSelectorTest.InstanceMethods (Introspection.MacApiSelectorTest.ApiSelectorTest.InstanceMethods)
```

Comment:
if it works then you can update the test with a comment "routes to concrete type" or whatever verbiage we have there for similar cases.